docs/src/main/sphinx/connector/object-storage-cache.md

```diff
@@ -0,0 +1,114 @@
+# Object storage filesystem caching
+
+Trino accesses files directly on object storage. This often involves the
+transfer of large amounts of data. The objects are retrieved from HDFS, or any
+other supported object storage, by multiple workers and processed on these
+workers. Repeated queries with different parameters, or even different queries
+from different users, often access, and therefore transfer, the same objects.
+
+Trino includes support for caching these objects with the help of the open
+source [Alluxio](https://github.com/Alluxio/alluxio) libraries with catalogs
+using the following connectors:
+
+* [](/connector/delta-lake)
+* [](/connector/hive)
+* [](/connector/iceberg)
```

Comment:
Current PR only supports delta lake. Iceberg and hive support are easy to add but need to be worked upon and are not covered by the current PR.

Reply:
Ok .. will adjust